src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Microsoft.DotNet.UpgradeAssistant.Cli.csproj

```diff
@@ -23,7 +23,8 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Autofac">
+      <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
Can System.Text.Json be used here instead?

Reply:
@twsouthwick The reason I added this becasue I've noticed a lot of exceptions during migration about NewtonSoft.Json 13.0.1 not being found. The CLI doesn't directly use it, but by adding an explicit package reference here the the latest version can now be found.

Reply:
makes sense